### DISCUSSION TITLE
Jdownloader 2

### DISCUSSION THREAD

**Discussion by Outshine8** [catppuccin/catppuccin#2538] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Hey :wave:,

Please add links or point towards examples on how this application can be themed.

**Comment by rgm89git** [comment]
Dracula has a JD2 theme. We can use it as a example:
[**Dracula's JD2 GitHub**](https://github.com/dracula/jdownloader2)

There is other examples like this Material Darker theme:
[**Material Darker GitHub**](https://github.com/moktavizen/material-darker-jdownloader/)

**_Previews for both themes:_**
![image](https://github.com/user-attachments/assets/aee5ce95-8a93-4819-b230-0e404e3d1619)
![image](https://github.com/user-attachments/assets/fbfb1091-42ac-441b-a499-0712e3abe31d)
